modules/gapi/test/cpu/gapi_imgproc_tests_fluid.cpp

```diff
@@ -15,50 +15,50 @@ namespace opencv_test
 {
 
 INSTANTIATE_TEST_CASE_P(RGB2GrayTestFluid, RGB2GrayTest,
-                        Combine(Values(ToleranceRGBBGR(0.001).to_compare_f()),
+                        Combine(Values(ToleranceColor(1e-3).to_compare_f()),
                                 Values(cv::Size(1280, 720),
                                 cv::Size(640, 480)),
                                 Values(true, false),
                                 Values(cv::compile_args(IMGPROC_FLUID))));
 
 INSTANTIATE_TEST_CASE_P(BGR2GrayTestFluid, BGR2GrayTest,
-                        Combine(Values(ToleranceRGBBGR(0.001).to_compare_f()),
+                        Combine(Values(ToleranceColor(1e-3).to_compare_f()),
                                 Values(cv::Size(1280, 720),
                                        cv::Size(640, 480)),
                                 Values(true, false),
                                 Values(cv::compile_args(IMGPROC_FLUID))));
 
 INSTANTIATE_TEST_CASE_P(RGB2YUVTestFluid, RGB2YUVTest,
-                        Combine(Values(ToleranceRGBBGR(0.15*3).to_compare_f()),
+                        Combine(Values(ToleranceColor(1e-3).to_compare_f()),
                                 Values(cv::Size(1280, 720),
                                        cv::Size(640, 480)),
                                 Values(true, false),
                                 Values(cv::compile_args(IMGPROC_FLUID))));
 
 INSTANTIATE_TEST_CASE_P(YUV2RGBTestFluid, YUV2RGBTest,
-                        Combine(Values(ToleranceTriple(0.25 * 3, 0.01 * 3, 1e-5 * 3).to_compare_f()),
+                        Combine(Values(ToleranceColor(1e-3).to_compare_f()),
                                 Values(cv::Size(1280, 720),
                                        cv::Size(640, 480)),
                                 Values(true, false),
                                 Values(cv::compile_args(IMGPROC_FLUID))));
 
 INSTANTIATE_TEST_CASE_P(RGB2LabTestFluid, RGB2LabTest,
-                        Combine(Values(ToleranceTriple(0.25 * 3, 0.0, 1e-5 * 3).to_compare_f()),
+                        Combine(Values(AbsSimilarPoints(1, 0.05).to_compare_f()),
                                 Values(cv::Size(1280, 720),
                                        cv::Size(640, 480)),
                                 Values(true, false),
                                 Values(cv::compile_args(IMGPROC_FLUID))));
 
 // FIXME: Not supported by Fluid yet (no kernel implemented)
-INSTANTIATE_TEST_CASE_P(BGR2LUVTestFluid, BGR2LUVTest,
-                        Combine(Values(ToleranceTriple(0.25 * 3, 0.01 * 3, 0.0001 * 3).to_compare_f()),
+INSTANTIATE_TEST_CASE_P(DISABLED_BGR2LUVTestFluid, BGR2LUVTest,
```

Comment:
Why is this test disabled?

Reply:
Perhaps because of the comment above.
And large absolute difference `=6`.

Re-enabled test, increased tolerance.

BTW, OpenCV CPU implementation is based on lookup tables / spline interpolations.

Reply:
OK, thanks a lot for the clarifications, and for re-enabling this test!